ml_report.tex

```diff
@@ -15,7 +15,7 @@
 \usepackage{tikz}
 \MakeOuterQuote{"}
 
-\usepackage[backend=biber,sorting=none, urldate=edtf, date=edtf]{biblatex}
+\usepackage[backend=bibtex,sorting=none, urldate=edtf, date=edtf]{biblatex}
```

Comment:
Are we sure that bibtex supports everything? It is an older version of biblatex
https://tex.stackexchange.com/questions/25701/bibtex-vs-biber-and-biblatex-vs-natbib#25702

Reply:
also this breaks the CI build

Reply:
it also breaks compiling on my machine, because I use the make file

Reply:
If we are to keep this then we need to change the makefile not to run biber (that will fix the CI error)